packages/client/components/MeetingControlBar.tsx

```diff
@@ -109,7 +109,12 @@ const MeetingControlBar = (props: Props) => {
   const [confirmingButton, setConfirmingButton] = useClickConfirmation()
   const cancelConfirm = confirmingButton ? () => setConfirmingButton('') : undefined
   const tranChildren = useTransition(buttons)
-  const {onMouseDown, onClickCapture} = useDraggableFixture(isLeftSidebarOpen, isRightDrawerOpen)
+  const showRightDrawer = isRightDrawerOpen && meetingType === MeetingTypeEnum.poker && phaseType === NewMeetingPhaseTypeEnum.ESTIMATE || false
```

Comment:
big picture,  if `isRightDrawerOpen` is true, I'd probably expect to see a right drawer somewhere. instead of managing 2 different pieces of state, let's just make this one always be the single source of truth.

in the estimate phase, add a hook that sets `isRightDrawerOpen` to true when the component mounts & sets it to false when the component unmounts. if entry animations are a pain, we can default it to `true` somewhere up higher if needed.